### PR TITLE
remove duplicate "Cost" i18n keys and create one chart.cost key

### PR DESCRIPTION
**Pull request by ddonahue007**
Issue: https://issues.redhat.com/browse/COST-727

Merge duplicate i18n keys into one key,value ("chart.cost": "Cost")

"Cost"                                                                     [Found: 11]
	FOUND IN KEY: gcp_details.cost_column_title
	FOUND IN KEY: aws_dashboard.cost_label
	FOUND IN KEY: azure_details.cost_column_title
	FOUND IN KEY: azure_dashboard.cost_label
	FOUND IN KEY: gcp_dashboard.cost_label
	FOUND IN KEY: ocp_cloud_dashboard.cost_label
	FOUND IN KEY: azure_cloud_dashboard.cost_label
	FOUND IN KEY: aws_cloud_dashboard.cost_label
	FOUND IN KEY: ocp_details.cost_column_title
	FOUND IN KEY: aws_details.cost_column_title
	FOUND IN KEY: ocp_details.total_cost

### CODE REVIEW

**Review by boaz0 — APPROVED** [review]

File: src/locales/en.json [hunk 1]
```diff
@@ -121,6 +115,7 @@
     "usage_cost_title": "Usage cost"
   },
   "chart": {
+    "cost": "Cost",
```
Comment (dlabrecq) [inline src/locales/en.json:118]: This is not a chart string.

"cost_label" is used in the dashboard cards, while "cost_column_title" is a table heading in details.

File: src/pages/views/details/awsDetails/detailsTable.tsx [hunk 2]
```diff
@@ -158,7 +158,7 @@ class DetailsTableBase extends React.Component<DetailsTableProps> {
             },
             {
               orderBy: 'cost',
-              title: t('aws_details.cost_column_title'),
+              title: t('chart.cost'),
```
Comment (dlabrecq) [inline src/pages/views/details/awsDetails/detailsTable.tsx:161]: The details table should not use strings associated with charts.

File: src/pages/views/details/ocpDetails/detailsHeader.tsx [hunk 3]
```diff
@@ -117,7 +117,7 @@ class DetailsHeaderBase extends React.Component<DetailsHeaderProps> {
             </Title>
             <div style={styles.costLabel}>
               <div style={styles.costLabelUnit}>
-                {t('ocp_details.total_cost')}
+                {t('chart.cost')}
```
Comment (dlabrecq) [inline src/pages/views/details/ocpDetails/detailsHeader.tsx:120]: We're showing a chart string in the details page header.

File: src/store/dashboard/awsCloudDashboard/awsCloudDashboardWidgets.ts [hunk 4]
```diff
@@ -19,7 +19,7 @@ export const computeWidget: AwsCloudDashboardWidget = {
   reportPathsType: ReportPathsType.awsCloud,
   reportType: ReportType.instanceType,
   details: {
-    costKey: 'aws_cloud_dashboard.cost_label',
+    costKey: 'chart.cost',
```
Comment (dlabrecq) [inline src/store/dashboard/awsCloudDashboard/awsCloudDashboardWidgets.ts:22]: This really doesn't have anything to do with charts, but the details section of the dashboard card

**Review by dlabrecq — CHANGES_REQUESTED** [review]

**Comment by dlabrecq** [comment]
Thinking we don't want to store the "Cost" string in the "chart" section. It really has nothing to do with charts.
Especially, when the string is used with table columns, page headings, etc.
Probably best to add "cost" at the top level, since it's a common string used in a bunch of places.

File: src/locales/en.json (hunk 1 above)
Comment (ddonahue007) [reply src/locales/en.json]: Done

**Review by dlabrecq — APPROVED** [review]

**Pull request merged**